notebooks/Time_Series_Forecasting.ipynb

```diff
@@ -0,0 +1,660 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Su5oVLIL-GK5"
+   },
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import sklearn\n",
+    "\n",
+    "%matplotlib inline\n",
+    "warnings.simplefilter(action='ignore', category=FutureWarning)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 237
+    },
+    "id": "1jUWQVXZ-HOs",
+    "outputId": "28317993-f1b2-44c1-d468-1fd00fb58f0f",
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "train_data = pd.read_csv('C:\\\\Users\\\\Owner\\\\Desktop\\\\Proga\\\\REPOS\\\\mentorship_EPAM\\\\data\\\\train.csv', index_col = \"id\")\n",
```

Comment:
It is not comfortble to specify full paths in your repository because of two points: they are long and compromise your local folder structure (username, in particular).
Also `\` path separator is windows-specific, to stay cross-platform you should either use [os](https://docs.python.org/3/library/os.html) or [pathlib](https://docs.python.org/3/library/pathlib.html) library. With these libraries you can easily specify and use the root data folder to reduce copy-pasting.

All in all, it would be much better to introduce a variable like
```
# from pathlib import Path
DATA_ROOT = Path('data')
```
and use it everywhere inside your notebook like
```suggestion
    "train_data = pd.read_csv(DATA_ROOT / 'train.csv', index_col = \"id\")\n",
```
**But**: to make this code working you need to make two changes:
1. Set temporary environment variable in `notebook.cmd`, like
```
# add these lines just before jupyter notebook command execution
setlocal
set PROJECT_ROOT=%cd%
```
2. In every notebook you develop add the following code to the very first cell:
```
import os
os.chdir(os.environ["PROJECT_ROOT"])
```
This will set your current working directory to the root of your project (inside the notebook).